api/modules/users/views.py

```diff
@@ -415,3 +415,25 @@ def generate_verification_code(request):
         error_message = "Unable to send verification code email to user"
         return Response(error_message, status=status.HTTP_400_BAD_REQUEST)
     return Response("Verification code sent", status=status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+@permission_classes((AllowAny,))
```

Comment:
This is not necessary, please remove this

Reply:
I think it's better indeed to request authentication to confirm the code. It's done as requested.   😉